django_mongodb/features.py

```diff
@@ -512,6 +517,7 @@ def django_test_expected_failures(self):
             "delete.tests.DeletionTests.test_only_referenced_fields_selected",
             "lookup.tests.LookupTests.test_in_ignore_none",
             "lookup.tests.LookupTests.test_textfield_exact_null",
+            "queries.test_qs_combinators.QuerySetSetOperationTests.test_exists_union",
```

Comment:
This test seems useful except for the SQL introspection part, so I comment out the unneeded lines in the latest commit on the Django fork.